java/core/src/test/org/apache/orc/TestTypeDescription.java

```diff
@@ -174,14 +174,6 @@ public void testQuotedField1() {
     assertTrue(e.getMessage().contains("Unmatched quote at 'struct<^`abc'"));
   }
 
-  @Test
-  public void testQuotedField2() {
-    IllegalArgumentException e = assertThrows(IllegalArgumentException.class, () -> {
-      TypeDescription.fromString("struct<``:int>");
-    });
-    assertTrue(e.getMessage().contains("Empty quoted field name at 'struct<``^:int>'"));
```

Comment:
This test coverage means it's a feature before.

Reply:
I'd probably prefer to just ignore it instead of removing it.